manifest_staging/charts/aad-pod-identity/templates/mic-deployment.yaml

```diff
@@ -155,6 +155,9 @@ spec:
                 key: UserAssignedMSIClientID
                 name: {{ template "aad-pod-identity.mic.fullname" . }}
           {{- end }}
+          {{- else if .Values.mic.customCloud.enabled }}
```

Comment:
This just needs to be an `if` instead of `else if`. It's possible to use custom cloud + admin secret.

Reply:
Ok, I updated. That meant I had to make the volumes and volumeMounts conditional on either not adminsecret or custom cloud, so it's a slightly bigger change than just this line.